mozilla/bedrock
# Family FAQ: update screentime answer (fixes #12735)

## One-line summary

Updates answer to be more specific in age-related guidance. See https://docs.google.com/document/d/1IKGtg6oNADrEIde4rv3XwXKopG3QyTBNnVcU29KxstU/edit# (mozilla only)

Aiming for Monday Feb. 13 go-live

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12735

## Testing

To test this work:

- [ ] http://localhost:8000/en-US/firefox/family/#faq
Open "How long is too long to stick my preschooler in front of the iPad?" and check copy matches google doc updates
